data.sql

```diff
@@ -9,3 +11,28 @@ VALUES ('Pikachu', '2021-01-07', 1, false, 15.04);
 
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES ('Devimon', '2017-05-12', 5, true, 11);
+
+
+/* PROJECT 2 - UPDATE TABLE WIOTH NEW ROWS */
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species)
+VALUES ('Charmander', '2020-02-08', 0, false, -11, 'Fire');
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species)
+VALUES ('Plantmon', '2021-11-15', 2, true, -5.7, 'Grass');
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species)
+VALUES ('Squirtle', '1993-04-02', 3, false, -12.13, 'Water');
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species)
+VALUES ('Angemon', '2005-06-12', 1, true, -45, 'Angel');
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species)
+VALUES ('Boarmon', '2005-06-07', 7, true, 20.4, 'Mammal');
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species)
+VALUES ('Blossom', '1998-10-13', 3, true, 17, 'Flower');
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species)
+VALUES ('Ditto', '2022-05-14', 4, true, 22, 'Unknown');
```

Comment:
I noticed some `species` in your table different from what is given in the [project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md) 👀

`Flower`, `Water`, and `Fire`

![image](https://github.com/jicamargo/vet_clinic/assets/78251927/2e5f8fc9-dc30-4ec3-a91e-bc607381e6d0)

As a result of this, the screenshot provided does not answer the question below
* What is the minimum and maximum weight of each type of animal?

In the requirements, there are two species, `Digimon` and `Pokemon`

![image](https://github.com/jicamargo/vet_clinic/assets/78251927/85cd4507-cea1-4b68-a7c5-79f9d89fc216)

Kindly update your table and include a screenshot of the results 🛠👍

Reply:
Thank you @abdulhamiid, I added some sample data and forgot to delete them before the final exercise.

I'll do the changed. 👍🏼